dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -401,138 +399,119 @@
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_MAX",
+                      "groupByFields": [
+                        "resource.label.\"project_id\"",
+                        "resource.label.\"location\"",
+                        "resource.label.\"cluster_name\"",
+                        "metric.label.\"container_name\"",
+                        "resource.label.\"namespace_name\"",
+                        "resource.label.\"controller_name\""
+                      ],
+                      "perSeriesAligner": "ALIGN_MAX"
+                    },
+                    "filter": "metric.type=\"kubernetes.io/autoscaler/container/memory/per_replica_recommended_request_bytes\" resource.type=\"k8s_scale\" resource.label.\"namespace_name\"!=monitoring.regex.full_match(\"gmp-system\") ${project_id} ${location} ${cluster_name} ${namespace_name}"
+                  }
                 }
               }
             ],
             "metricVisualization": "NUMBER"
-          },
-          "title": "Memory request MiB recommended per replica"
-        },
-        "width": 6,
-        "xPos": 0,
-        "yPos": 16
+          }
+        }
       },
       {
-        "height": 4,
+        "xPos": 24,
+        "yPos": 60,
+        "width": 24,
+        "height": 16,
         "widget": {
+          "title": "CPU request mCore recommended per replica",
           "timeSeriesTable": {
             "columnSettings": [
-              {
-                "column": "resource.project_id",
-                "visible": true
-              },
-              {
-                "column": "resource.location",
-                "visible": true
-              },
-              {
-                "column": "resource.cluster_name",
-                "visible": true
-              },
-              {
-                "column": "resource.namespace_name",
-                "visible": true
-              },
-              {
-                "column": "resource.controller_name",
-                "visible": true
-              },
-              {
-                "column": "resource.controller_kind",
-                "visible": true
-              },
-              {
-                "column": "metric.container_name",
-                "visible": true
-              },
               {
                 "column": "value",
-                "visible": true
+                "visible": false
```

Comment:
Agree, let's delete the "columnSettings" field for simplicity, because the table widget is essentially defaulting to showing columns for all the group-bys anyway.
![Screenshot 2024-01-31 at 9 18 26 AM](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/assets/6540967/0a3acb5a-7ea6-4522-a14b-7d4d5b7f1bd8)

Reply:
Same applies for "Memory request MiB recommended per replica"

Reply:
I updated this removing the columnSettings